tools/psoxy-test/lib/gcp.js

```diff
@@ -247,7 +247,7 @@ async function download(bucketName, fileName, destination, client, logger) {
   }
 
   const downloadFunction = async () => client.bucket(bucketName).file(fileName)
-    .download({ destination: destination });
+    .download({ destination: destination, decompress: false });
```

Comment:
as-is, was failing due to auto-magic decompress by GCS when we properly set `Content-Encoding`, which we do now (didn't before)